### ISSUE TITLE
make sure all projects have xtext nature

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-core#18]

**Comment by cdietrich** [comment]
can gradle do that?

**Comment by cdietrich** [comment]
the following does the trick, but where to put it? @meysholdt @spoenemann do you have an idea

```
eclipse {
	project {
		natures 'org.eclipse.xtext.ui.shared.xtextNature'
		buildCommand 'org.eclipse.xtext.ui.shared.xtextBuilder'
	}	
}
```

**Comment by cdietrich** [comment]
see https://github.com/eclipse/xtext-extras/issues/19 as well

**Comment by spoenemann** [comment]
@cdietrich that can go to `gradle/eclipse-project-layout.gradle`

**Comment by cdietrich** [comment]
fixed with e3c6f93b3c35db1a92aaa356063cbaa384e1212d
